homeassistant/components/xiaomi_miio/number.py

```diff
@@ -68,7 +68,7 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
     if model not in [MODEL_AIRHUMIDIFIER_CA4]:
         return
 
-    for feature in NUMBER_TYPES:
+    for feature in NUMBER_TYPES.items():
```

Comment:
This is not correct. `items()` returns tuples `(key, value)` while `NUMBER_TYPES` is indexed with a single value:

https://github.com/jbouwh/core/blob/3bbcd3e4b22579f2e7800b8db7d80654092e0789/homeassistant/components/xiaomi_miio/number.py#L43

The current behavior is correct, as it iterates over the keys.

Reply:
Oh indeed 😓  well spotted, thanks